tools/run_tests/dockerize/docker_run_tests.sh

```diff
@@ -42,7 +42,21 @@ export PATH=$PATH:/usr/bin/llvm-symbolizer
 chown $(whoami) $XDG_CACHE_HOME
 
 mkdir -p /var/local/git
-git clone --recursive /var/local/jenkins/grpc /var/local/git/grpc
+git clone /var/local/jenkins/grpc /var/local/git/grpc
+
+# (todo (mattkwong): the /var/local/jenkins/grpc has no submodules and boringssl 
```

Comment:
where did you see "has no submodules and boringssl has non-submodule files in it"?

Action for cloning submodules can be set in Jenkins UI (separately for each job), but docker_run_tests.sh is used for running run_tests.py and I'd be surprised if we didn't setup the submodule cloning for some of the jobs. In any case, if it happens, it's jenkins job specific I think.